_commands/tools/file.md

```diff
@@ -0,0 +1,42 @@
+---
+---
+
+file
+--
+
+`file` is used to find out the _type_ of a file
+
+~~~ bash
+$ file file.md
+file.md: ASCII text
+~~~
+
+### Useful Options/Examples
+
+### `file -F, --separator _separator_`
```

Comment:
I think I understand that the `,` is trying to separate the command flags from the command arguments, but it's not obvious that this is the case. I would rather them just be removed- I think it's clear enough without them.

Additionally, the redundancy of the term "separator" twice (`--seperator`, `_separator_`), but only referring to one instance of it (`_separator_`) is also a little confusing. I just want to make sure the first instance (`--separator`) isn't a type, because it doesn't look like you include the word after the two dashes in the code example below.